libs/libyubikey/Makefile

```diff
@@ -2,7 +2,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libyubikey
 PKG_VERSION:=1.13
-PKG_RELEASE:=1
```

Comment:
Consider using `$(AUTORELEASE)` if you feel fancy.

Reply:
I don't. Let's leave that up to the maintainer?